instagram/src/components/Login/login.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = { username: "", password: "" };
+  }
```

Comment:
interestingly enough, since you don't pass `props` in to the constructor or super here, you can leave the constructor out entirely and set state in the component with a simple declaration: `state={username, "", password:""};`